internal/sirius/team.go

```diff
@@ -39,18 +39,21 @@ func (c *Client) Team(ctx context.Context, cookies []*http.Cookie, id int) (Team
 	team := Team{
 		ID:          v.Data.ID,
 		DisplayName: v.Data.DisplayName,
-		Type:        "LPA",
+		Type:        "",
+		Email:       v.Data.Email,
+		PhoneNumber: v.Data.PhoneNumber,
 	}
 
 	for _, m := range v.Data.Members {
 		team.Members = append(team.Members, TeamMember{
+			ID:          m.ID,
 			DisplayName: m.DisplayName,
 			Email:       m.Email,
 		})
 	}
 
 	if v.Data.TeamType != nil {
-		team.Type = "Supervision — " + v.Data.TeamType.Label
+		team.Type = v.Data.TeamType.Handle
```

Comment:
Would it be better to get both of these instead of using a static `map` (`Type` and `TypeLabel`?)

Reply:
Or get the map from `/api/v1/reference-data?filter=teamType` on startup

Reply:
We only display the type label when pulling from `Teams()`, which is why I haven't set it here, but I guess there's no reason not to and it would be a bit less surprising.

Good pull on the endpoint, I didn't know that was a thing! I'll replace my static `TeamTypeOptions` map with a new endpoint and call it when generating the edit form.

Reply:
I could probably have done something similar for the user roles when I hardcoded that list in html